### PR TITLE
Add pgp public info page

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

zeplin: https://zpl.io/Z1Qmmq7

### CODE REVIEW

**Comment by MarcoPolo** [comment]
The inputs are a bit hard to style, so for the full name I didn't quite match it pixel perfect to the design.

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits e2a10037aa1d...c09f84af858d introduced some visual changes on linux:
 * Added: **Add_Pgp-One_Error_Email**  
   Add_Pgp-One_Error_Email rendered
 * Added: **Add_Pgp-Normal**  
   Add_Pgp-Normal rendered
 * Added: **Add_Pgp-Empty**  
   Add_Pgp-Empty rendered

File: shared/profile/pgp/add.desktop.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+type SmallInputProps = {
+  hintText: string,
+  label: string,
+  value: ?string,
+  onChange: (next: string) => void,
+  errorState: boolean,
+}
+
+function SmallInput ({hintText, label, onChange, value, errorState}: SmallInputProps) {
```
Comment (chrisnojima) [inline shared/profile/pgp/add.desktop.js]: These will be used elsewhere so we should hoist it to common adapters i think

File: shared/profile/pgp/add.desktop.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  )
+}
+
+class PgpAdd extends Component<void, Props, void> {
+  render () {
+    const nextDisabled = !this.props.email1 || !this.props.fullName
+    return (
+      <Box style={containerStyle}>
+        {/* TODO(MM) when we get the pgp icon, put it in here */}
+        <Box style={{backgroundColor: 'black', width: 48, height: 48, alignSelf: 'center'}} />
```
Comment (chrisnojima) [inline shared/profile/pgp/add.desktop.js]: if we have this situation lets use <Icon with a type like 'confont-identity-pgp'

File: shared/profile/pgp/dumb.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import PgpAdd from './add'
```
Comment (chrisnojima) [inline shared/profile/pgp/dumb.js]: flow

File: shared/profile/pgp/dumb.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    'Normal': addBase,
+    'Empty': {
+      ...addBase,
+      email1: null,
+      fullName: null,
+    },
+    'One Error - Email': {
+      ...addBase,
+      errorText: 'Some email addresses are invalid. Please fix!',
+
```
Comment (chrisnojima) [inline shared/profile/pgp/dumb.js]: 🔥

**Comment by chrisnojima** [comment]
👍 % 💬

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits e2a10037aa1d...206c5c5d7599 introduced some visual changes on linux:
 * Added: **Add_Pgp-One_Error_Email**  
   Add_Pgp-One_Error_Email rendered
 * Added: **Add_Pgp-Normal**  
   Add_Pgp-Normal rendered
 * Added: **Add_Pgp-Empty**  
   Add_Pgp-Empty rendered

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits e2a10037aa1d...62b602e71f9e introduced some visual changes on linux:
 * Added: **Add_Pgp-One_Error_Email**  
   Add_Pgp-One_Error_Email rendered
 * Added: **Add_Pgp-Normal**  
   Add_Pgp-Normal rendered
 * Added: **Add_Pgp-Empty**  
   Add_Pgp-Empty rendered

**Pull request merged**